content/posts/Introductory-GSoC2020-post/index.md

```diff
@@ -0,0 +1,51 @@
+
+I also have a keen interest in Machine Learning and Natural Language Processing and have done a couple of projects at my university. I have researched on `Query Expansion using fuzzy logic`. I will be publishing it in some time. It involves the fuzzification of the traditional wordnet for query expansion. 
+
+Our paper `Experimental Comparison & Scientometric Inspection of Research for Word Embeddings` got accepted in ESCI Journal and Springer LNN past week.  It explains the ongoing trends in universal embeddings and compares them.
+
+Getting started with matplotlib
+
+I chose matplotlib as it is an organization with so much cool stuff relating to plotting. I have always wanted to work on such things. People are really friendly, always eager to help!
+
+## Taking Baby steps:
+
+The first step is getting involved with the community. I started using the Gitter channel to know about the maintainers. I started learning the different pieces which tie up for the baseline image problem. I started with learning the system architecture of matplotlib. Then I installed the matplotlib, learned the cool tech stack related to matplotlib like sphinx, python, pypi etc.
+
+## Keep on contributing and keep on learning:
+
+Learning is a continuous task. Taking guidance from mentors about the various use case scenarios involved in the GSoC project helped me to gain a lot of insights. I solved a couple of small issues. I learned about the code-review process followed here, sphinx documentation, how releases work. I did [some PRs](https://github.com/matplotlib/matplotlib/pulls?q=is%3Apr+author%3ASidharthBansal+is%3Aclosed). It was a great learning experience.   
+
+## About the Project:
+
+[The project](https://github.com/matplotlib/matplotlib/issues/16447) is about the generation of baseline images instead of downloading them. The baseline images are problematic because they cause the repo size to grow rather quickly by adding more baseline images. Also, the baseline images force matplotlib contributors to pin to a somewhat old version of FreeType because nearly every release of FreeType causes tiny rasterization changes that would entail regenerating all baseline images. Thus, it causes even more repository size growth. 
+The idea is not to store the baseline images at all in the Github repo. It involves breaking up the matplotlib package into mpl-test and mpl-notest. Then creating logic for generating and grabbing the latest release. We will be requiring some caching too. We will then implement an analogous strategy to the CI.  
```

Comment:
Slightly concerned about what "breaking up the matplotlib package into mpl-test and mpl-notest" is claiming.

Reply:
I have added two lines to explain it. Thanks for pointing out.